ntbs-service/Pages/Shared/_NotificationLayout.cshtml

```diff
@@ -18,15 +19,15 @@
             break;
     }
 
-    ViewData["IsOverviewPage"] = currentPage == "Overview";
+    ViewData["IsOverviewPage"] = currentPage == string.Empty;

```

Comment:
+Christian comment questioning whether this will be matched in all instances, and maybe whether we reassess this approach completely.